src/components/CreateManifestForm/CreateManifestForm.tsx

```diff
@@ -154,20 +86,37 @@ const CreateManifestForm: FC<CreateManifestFormProps> = (props) => {
             isRequired
             fieldId="create-satellite-manifest-form-name"
           >
-            <TextInput
-              isRequired
-              type="text"
-              id="create-satellite-manifest-form-name"
-              value={name}
-              placeholder="Name"
-              onChange={handleNameChange}
-              onBlur={validateName}
+            <Controller
+              name="satelliteManifestName"
+              control={control}
+              defaultValue=""
+              rules={{
+                required: 'Please provide a name for your new manifest',
+                maxLength: {
+                  value: 99,
+                  message: 'Your manifest name must be less than 100 characters'
+                },
+                pattern: {
+                  value: /^[0-9A-Za-z_.-]+$/,
+                  message: `Your manifest name must use 
```

Comment:
How about "may contain", like this:
"Your manifest name may contain only numbers, letters, underscores, hyphens, and periods."

Reply:
ok just updated